[8.0] account_move_line_report_xls description update

luc-demeyer opened the pull request.

luc-demeyer committed:
description update

pedrobaeza commented:
:+1: 

Please check https://github.com/OCA/account-financial-reporting/pull/63 also for an updated icon.

Thanks

yvaucher commented:
:+1:

lmignon commented on account_move_line_report_xls/README.rst:
```diff
@@ -7,10 +7,10 @@ This module extends the functionality of the journal items
 Installation
```
Would be nice to also have the licence badge on top of the description...

``` rest
.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
    :alt: License
```

luc-demeyer committed:
update icon aml_report_xls, fix travis error account_financial_report_webkit

luc-demeyer commented:
I have updated the PR with the new icon as suggested by Pedro.
At the same time I also fixed the travis error in the account_financial_report_webkit module.

tafaRU commented:
Would be nice also use  https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst but the rest LGTM :+1:

luc-demeyer committed:
update README.rst

luc-demeyer commented:
@nbessi 
Travis fails because of broken yml test (aged_trial_balance.yml).
Can you have a look ?

lmignon commented:
:+1:

tafaRU commented:
@luc-demeyer please ignore my previous comment, diff have measled me :wink:

nbessi commented:
@luc-demeyer hmm I'm not sure of the root cause. Maybe the default for period_from is not call.
It should take some time to debug.

luc-demeyer commented:
@nbessi can this PR in the meanwhile be merged since travis error is independent from the PR ?

nbessi commented:
@luc-demeyer I have no objection but it will report the problem later. At  least a bug report must be opened

luc-demeyer commented:
@nbessi cf. https://github.com/OCA/account-financial-reporting/issues/67

yvaucher commented:
Please rebase. Your PR shouldn't change icon as it was already done in #63

luc-demeyer commented:
@yvaucher 
Rebase done. Thanks for remarking this.

yvaucher commented:
@luc-demeyer Thanks for the change :+1: 

We can merge now
